### PR TITLE
Adding structure to instructor guides/notes

### PR DESCRIPTION
**Pull request by k8hertweck**
I've tried my hand at a draft of an instructor guide template. I've made the assumption that the guide should be a supplement to the lesson, rather than required reading (e.g, lessons should be stand-alone, for both instructor and learner use). I've tried to make this as generalizable to the whole set of DC/SWC lessons as possible (with the hope that DC adopts the new SWC template, but this certainly isn't required to use the language in this instructor guide template). This structure seems to be relatively easy to adopt into existing instructor guides, and would also help ease the way for new instructor guides to be developed.

This template is relatively simple, but I would be in favor of adding more structure to the section on lesson design and am open to any ideas!

### CODE REVIEW

**Comment by gvwilson** [comment]
Thanks for doing this - :+1: from me. Mail the maintainers and get them to weigh in?

**Comment by k8hertweck** [comment]
Hoping for feedback from mentors this afternoon; will then message maintainers for comments/concerns.

**Comment by gvwilson** [comment]
Any feedback from maintainers, or should we just go ahead and merge?

**Comment by naupaka** [comment]
I think this is great! Just noticed it now. Big +1 from me.

**Comment by gdevenyi** [comment]
+1

File: _extras/guide.md [hunk 1]
```diff
@@ -1,8 +1,40 @@
 ---
 layout: page
-title: "Instructors' Guide"
+title: "Instructor Notes"
 permalink: /guide/
```
Comment (gdevenyi) [inline _extras/guide.md]: Should the permalink be changed? Does this require a template change somewhere as well?

Comment (k8hertweck) [inline _extras/guide.md]: I considered changing the permalink, but after discussions with the mentoring subcommittee we decided the easiest course of action would be to leave the permalink/file names as guide.

**Pull request merged**